app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,8 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    version = ext_management_system.highest_supported_api_version
+    version >= '4.0' if version
```

Comment:
Isn't `VmImport` included into `ExtManagementSystem`?  I don't see how this is related to a VM being disconnected.